modules/ocf_kubernetes/templates/ingress/ingress_expose.yaml.erb

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+  labels:
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+spec:
+  type: NodePort
+  externalIPs:
+    <% @kubernetes_workers_ipv4.each do |ip| %>
+    - <%= ip %>
+    <% end %>
+  ports:
+    - name: http
+      port: 80
+      targetPort: 80
+      protocol: TCP
+    - name: https
+      port: 443
+      targetPort: 443
```

Comment:
If we're doing HTTPs through haproxy (which we probably should), then this isn't necessary.